docs/src/main/sphinx/admin/preview-web-interface.md

```diff
@@ -0,0 +1,29 @@
+# Preview Web UI
+
+In addition to the [](/admin/web-interface), Trino includes a preview version of
+a new web interface. It changes look and feel, available features, and many
+other aspects. In the future this new user interface will replace the existing
+user interface.
+
+:::{warning}
+The Preview Web UI is not suitable for production usage, and only available for
+testing and evaluation purposes. Feedback and assistance with development is
+encouraged.
```

Comment:
<3

Reply:
I'd add a link to the slack or whatnot to encourage colaboration

Reply:
We have never done that in the docs .. but I dont see why not.